yt_dlp/extractor/zdf.py

```diff
@@ -28,11 +29,11 @@ class ZDFBaseIE(InfoExtractor):
     def _call_api(self, url, video_id, item, api_token=None, referrer=None):
         headers = {}
         if api_token:
-            headers['Api-Auth'] = f'Bearer {api_token}'
+            headers['Api-Auth'] = 'Bearer %s' % api_token
```

Comment:
Revert all fstring changes

Reply:
done - a bit strange that hatch introduced them, but maybe I was using a wrong release of it, after installing it again with python 3.13.1 it no longer converts them to f strings